src/extension.ts

```diff
@@ -10,6 +10,9 @@ import getSHAArray from "./utils/getSHAArray";
 import setLoggedIn from "./utils/vscode/setLoggedIn";
 import getLocalUser from "./utils/vscode/getLocalUser";
 import getRepoInfo from "./utils/vscode/getRepoInfo";
+import request = require('request');
```

Comment:
Why is this import orphaned?
I don't see it referenced anywhere.

Reply:
I resolved because I removed it locally and you won't see it on the next push

Reply:
Thanks!